components/brave_wallet_ui/components/desktop/lock-screen/index.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react'
+
+import {
+  StyledWrapper,
+  Title,
+  IconBackground,
+  PageIcon,
+  InputColumn,
+  Input
+} from '../wallet-onboarding/create-password/style'
+import { NavButton } from '../../extension'
+import locale from '../../../constants/locale'
+
+export interface Props {
+  onSubmit: () => void
+  onPasswordChanged: (event: React.ChangeEvent<HTMLInputElement>) => void
```

Comment:
can we have this pass a simple parameter, e.g. `(value: string) => void`?

Reply:
Done!